CMakeLists.txt

```diff
@@ -16,34 +16,35 @@
 #
 ###############################################################################
 
-cmake_minimum_required (VERSION 3.6)
-project (YARPGen)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Do we really need 3.13? TravisCI fails because it has 3.12 for Linux and 3.11 for Mac.